tests/testthat/test-quiet.R

```diff
@@ -95,6 +112,9 @@ test_that("Using quiet supresses messages as expected", {
         })
     })
 
+    skip_if(Sys.getenv('USER') %in% c('secrestm','gowerc','sabanesd'))
+    skip_on_cran()
+    skip_on_ci()
```

Comment:
Is skip on CI needed? My understanding is that skip on cran would be sufficient ?

Reply:
@gowerc , yeah let's try in the next commit.

Reply:
@gowerc looks like that worked.